tests/unit/protocol-notification-bar.js

```diff
@@ -0,0 +1,64 @@
+/* global describe, beforeEach, it, spyOn */
+
+describe('protocol-notification.js', function() {
+
+    'use strict';
+
+    beforeEach(function () {
```

Comment:
Nit: Suggest adding an `afterEach` below here to remove the button after each test also?

Reply:
Ah! Yes! Good catch!